test/components/CalendarDay_spec.jsx

```diff
@@ -75,6 +76,23 @@ describe('CalendarDay', () => {
         expect(wrapper.prop('aria-label')).to.equal('chooseAvailableDate text');
       });
 
+      it('is formatted with the dateIsSelected phrase function when day is selected', () => {
+        const selectedModifiers = new Set(['selected', 'selected-start', 'selected-end']);
+
+        selectedModifiers.forEach((selectedModifier) => {
+          const modifiers = new Set().add(selectedModifier);
```

Comment:
`new Set([selectedModifier])`?

Reply:
Yep that works fine and I can change it, the use of `new Set().add(<whatever>);` is used heavily in this file so I was following in kind.

Reply:
fair, then it's fine to keep it :-)